stable/README.md

```diff
@@ -146,13 +147,14 @@ helm install nuodb-insights/insights --generate-name -n nuodb \
 
 ## Accessing NuoDB Insights
 
-By default Grafana will be available within the Kubernetes cluster via ClusterIP service. One way to access Grafana dashboards is to use port forwarding and navigate to http://localhost:8080/.
+By default, the NuoDB Insights Grafana dashboard WebUI will be available within the Kubernetes cluster via ClusterIP service. One way to access the WebUI dashboards is to use port forwarding and navigate your web browser to http://localhost:8080/.
 
 ```
-kubectl port-forward service/<release-name>-grafana 8080:80
+kubectl port-forward service/<name>-grafana 8080:80
 ```
+For `<name>`, use the value used in your `helm install` command for the `name` argument.
```

Comment:
you picked `insights`. We should just go with that.

Reply:
Sounds good, goes along with the simplify theme : ) 
Done.